### PR TITLE
[CAI-24] Updating Sponsors & Welcome page

### PR DESCRIPTION
**Pull request by aameen79**
Updating Sponsors & Welcome page

### CODE REVIEW

**Comment by aameen79** [comment]
@MikeRosTX @phrawzty kindly need your support

File: content/events/2024-cairo/welcome.md [hunk 1]
```diff
@@ -4,21 +4,33 @@ Type = "welcome"
...
.../eyESln0mAs+9A8wFfwhG4dQjjBtpaGhoaGiccEBDegU4I97ExqEUCpqkZiOtGm8OIUxG483hh+Kp0+XEyPc3bBQHqw/JoYrWUI/GnZtzrXpvjZg5qxKKwBQoAu5ciKL2zYVgmpSiM62kVFQtWiK2bNsmag+3blALaawFl4GjwNvAttZja2hoaGhoaKQCNKpXgzPdNlcimULRsmHn8AKVgvheGFy2uXzFSjmBkhMpEaY4fLhWbN6yVSxEWNOml0pFhOxYL0SxyJX3ThUVFbPFmtVrxD4oKA1tTBxFOnaBBeDPcDoI1BMqNTQ0NDQ0uhpoaDukUCRSDT2wx2FqSakoLZsJ9+nSjTt7tkeBIBmvvDe/SCojVYsWiy1btonDNW32QhwBV4BjwDvg1FGDNRoaGhoaGhqpAo3uUQpFDRrtRYuXxnsF3LkJfo1+MlJ5aK8CQf9URlQPBodF3ntvteyF4EZerQHp3wNOAX8J5sDp+FkV09DQ0NDQOBWBxvdaNMKVsmX2gEMJ+w8eFOvWvy9mz5mHRj/e2yDnPLRTWUhGpUQw3GklM8SChYvkEMmhtudC1IGrwfHgnSCtrmloaGhoaGgcL6B9/hAa5HfAVltxudnW9h1i8ZKlYnppWZMiIHsviv0VhkRSgaB/1QtRXlEpVq5aLfbs3Svq2u6F2A+WgL8GL4XTCbuBk4aGhoaGxikJNM60TxEFfwBOBHfIVjwJZO/F/v1i7bp10lhUIXsvoGD49V5QcaB7PpSPqdNK5DJRbhPO7cJbA9JQD64Faer683DiHhkaGhoaGhoaPQVowGldk8sqR4Kc4Niq0Qn2Xmzdtl0u35w2fYZULvLcXoiyilli+cpVYteutnf5RDwHwZng/eBHwI5u9qKhoaGhoaFxIgHtvLKu+VOQyy93x5t/f1Bp2Lt3n9ixY6c4cJDWNZMv60RY3GzrffBl8MsgdxvU0NDQ0NDQONkBPaA/Gv7bwafAlWDr3Q4JgP9qcBb4J5BGtbgtroaGhoaGhsapCugHqvfi52AxuCeuNjQDbsQG8FXwa2AIznq3RA0NDQ0NDQ1/QFkYAN4FPgfSLsRQ8GooEOe7XjQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0TgKkpf1/9e6lmzpyKpsAAAAASUVORK5CYII=" alt="" width="100%" height="39"></td>
```
Comment (nimbinatus) [inline content/events/2024-cairo/welcome.md:19]: Hi there! Is there a reason these images are all base64 encoded instead of standard JPGs or PNGs? Doing this disables caching and can cause issues depending on what you're trying to do.

Comment (aameen79) [reply content/events/2024-cairo/welcome.md:19]: The reason that I find difficulties to refer to external JPGs or PNGs, so I prefer having them in the code.
Kindly we need to publish urgently, and you can guide me to make the modifications as an enhancement refactoring step. @nimbinatus

**Comment by aameen79** [comment]
@MikeRosTX  @phrawzty  kindly need your support

**Comment by phrawzty** [comment]
Hello @aameen79. Some information:
1. `@MikeRosTX` is no longer handling PRs on this repo. Please do not ping him directly going forward.
2. @nimbinatus _is_ handling PRs on this repo. Kindly respect her observations and assistance going forward.
3. In general, you do not need to ping anybody directly for a straightforward PR review. Please do _not_ do that going forward.

Concerning this PR, unfortunately, these base64-encoded images should not have been [merged](https://github.com/devopsdays/devopsdays-web/pull/14344) in the first place. This was a mistake that was easy to make because:
1. Large diffs are not visualised in GitHub by default, and are easy to miss.
2. You did not explain, or even mention, the contents of the large diff.

In summary, errors were made on both sides, and the path forward is as follows:
1. This PR will be closed.
2. You will prepare another PR wherein:
  - The base64-encoded files are removed.
  - These logos are proper sponsors, using whatever sponsor level is appropriate for your event.
4. We will review the PR and merge.

@nimbinatus will provide additional guidance regarding sponsors, image formats, etc.

We remain at your disposal for any further assistance you may require.
